shaders/src/main/glsl/samples/320es/stable_maze.frag

```diff
@@ -100,11 +140,15 @@ void main() {
       }
     }
     if (map[ipos.y * 16 + ipos.x] == 1) {
+      // Early out: our pixel is in a wall
       _GLF_color = vec4(1.0, 1.0, 1.0, 1.0);
       return;
     }
   }
   while (canwalk);
-
+  
+  // If we manage to get here, there's no legal moves left,
+  // and our pixel is in a non-wall tile.
   _GLF_color = vec4(0.0, 0.0, 0.0, 1.0);
 }
+

```

Comment:
Similar comment to the above if this is an extra line (as opposed to an EOF newline that was missing).

Reply:
And similarly, it was the missing newline.